docs/README.md

```diff
@@ -27,42 +27,27 @@ You can download the installer for the *Recommended Build* or the
  
 For support and questions, visit [the NeoForged Discord server][Discord].
 
-# Creating Mods
+## Creating Mods
 
 [See the "Getting Started" section in the NeoForged Documentation][Getting-Started].
 
-# Contribute to Forge
+## Contribute to NeoForge
 
-If you wish to actually inspect Forge, submit PRs or otherwise work
- with Forge itself, you're in the right place!
-
- [See the guide to setting up a Forge workspace][ForgeDev].
+If you wish to actually inspect NeoForge, submit PRs or otherwise work
+with NeoForge itself, you're in the right place!
 
 ### Pull requests
 
-[See the "Making Changes and Pull Requests" section in the Forge documentation][Pull-Requests].
-
 Please read the contributing guidelines found [here][Contributing] before making a pull request.
 
 ### Contributor License Agreement
-We require all contributors to acknowledge the [Neoforged Contributor License Agreement][CLA]. 
+We require all contributors to acknowledge the [NeoForged Contributor License Agreement][CLA]. 
 Please ensure you have a valid email address associated with your GitHub account to do this. If you have previously 
  signed it, you should be OK.
 
-#### Donate
-*NeoForged is a large project with many collaborators working on it around the clock. It will always remain 
-free to use and modify. However, it costs money to run such a large project as this, so please consider visiting
-our opencollective.*
-
-[Contributing]: ./CONTRIBUTING.md
-[CLA]: https://cla-assistant.io/MinecraftForge/MinecraftForge
-
-[Download]: https://maven.neoforged.net/#/releases/net/neoforged/forge/
```

Comment:
Because of the removal of this link, the `[Download]` link up at the top (in the list under the support table) is now not a link. Please re-add this, probably best as a link to https://neoforged.net/ (since that's where the download links are at present).